tests/nested/manual/fakeinstaller/task.yaml

```diff
@@ -0,0 +1,165 @@
+summary: Check that the /system/<label> API works via fake installer
+
+# this is a UC20+ specific test
+systems: [ubuntu-20.04-64, ubuntu-22.04-64]
```

Comment:
(nitpick) we could limit to run on 22.04 for now as this is the target (but only if there are issues with 20.04 I guess that we don't want to fix right now)

Reply:
Right, at the moment there is no reason why the test would not work for 20, so I guess we can leave it as it is for the moment - if it stops working we can remove it with the advantage that we will find out what is missing on 20.04.